docs/docs/illixr_plugins.md

```diff
@@ -2,290 +2,641 @@
+
+Topic details:
+
+-   *Publishes* [`binocular_cam_type`][63] to `cam` topic.
+
+&nbsp;&nbsp;**Details**&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C11]
+
+## offline_imu
+
+Reads [_IMU_][36] data files on disk, emulating a real sensor on the [_headset_][38]
+(feeds the application input measurements with timing similar to an actual IMU).
+
+Topic details:
+
+-   *Publishes* [`imu_type`][64] to `imu` topic.
+
+&nbsp;&nbsp;**Details**&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C12]
+
+## offload_data
+
+Writes [_frames_][34] and [_poses_][37] output from the [_asynchronous reprojection_][35] plugin to disk for analysis.
+
+Topic details:
+
+-   Synchronously *reads* `texture_pose` to `texture_pose` topic.
+
+&nbsp;&nbsp;**Details**&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C13]
+
+## offload_rendering_client
+
+Receives encoded frames from the network, sent by [offload_rendering_server](#offload_rendering_server)
+
+Topic details:
+
+-   *Calls* [`vulkan::display_provider`][30]
+-   *Calls* [`pose_prediction`][32]
+-   Asynchronously *reads* `compressed_frame` from `compressed_frames` topic.
+-   *Publishes* [`fast_pose_type`][60] to `render_pose` topic.
+
+&nbsp;&nbsp;[**Details**][D14]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C14]
+
+## offload_rendering_client_jetson
+
+Receives encoded frames from the network, sent by [offload_rendering_server](#offload_rendering_server), but specialized to run on a Jetson.
+
+Topic details:
+
+-   *Calls* [`vulkan::display_provider`][30]
+-   *Calls* [`pose_prediction`][32]
+-   Asynchronously *reads* `compresswed_frame` from `compressed_frames` topic.
+-   *Publishes* [`fast_pose_type`][60] to `render_pose` topic.
+
+&nbsp;&nbsp;[**Details**][D15]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C15]
+
+## offload_rendering_server
+
+Encodes and transmits frames to one of the offload_rendering_clients. 
+
+Topic details:
+
+-   *Calls* [`vulkan::display_provider`][30]
+-   Asynchronously *reads* [`fast_pose_type`][60] from `render_pose_` topic.
+-   *Publishes* `compressed_frame` to `compressed_frames` topic.
+
+&nbsp;&nbsp;[**Details**][D16]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C16]
+
+## offload_vio
+
+Four plugins which work in unison to allow head tracking (VIO) to be rendered remotely. 
+
+Topic details:
+
+- `offload_vio.device_rx`
+  - Asynchronously *reads* a string from topic `vio_pose`.
+  - *Publishes* [`pose_type`][61] to `slow_pose` topic.
+  - *Publishes* [`imu_integrator_input`][66] to `imu_integrator_input` topic.
+- `offload_vio.device_tx`
+  - Asynchronously *reads* [`binocular_cam_type`][63] from `cam topic`
+  - *Publishes* a string to `compressed_imu_cam` topic
+- `offload_vio.server_rx`
+  - Asynchronously *reads* a string from `compressed_imu_cam` topic
+  - *Publishes* [`imu_type`][64] to `imu` topic.
+  - *Publishes* [`binocular_cam_type`][63] to `cam` topic.
+- `offload_vio.server_tx`
+  - Asynchronously *reads* [`imu_integrator_input`][66] from `imu_integrator_input` topic.
```

Comment:
Add `Synchronously *read* pose_type from slow_pose topic of open_vins`

Reply:
fixed